roles/splunk_common/tasks/java_tasks/install_oracle8_jdk.yml

```diff
@@ -57,3 +57,34 @@
   become: yes
   become_method: sudo
   become_user: root
+
+- name: Create splunk-launch.conf
+  become: yes
+  become_method: sudo
+  become_user: root
+  command: cp {{splunk.home}}/etc/splunk-launch.conf.default {{splunk.home}}/etc/splunk-launch.conf
+  ignore_errors: true
+  when:
+    - ansible_distribution == 'Debian'
+
+- name: Get JAVA_HOME
+  shell: 'readlink -f /usr/bin/java | sed "s:bin/java::"'
+  register: java_home
+  become: yes
+  become_method: sudo
+  become_user: root
+  ignore_errors: true
+  when:
+    - ansible_distribution == 'Debian'
+
+- name: Set JAVA_HOME in splunk-launch.conf
+  become: yes
+  become_method: sudo
+  become_user: root
+  lineinfile:
+    path: "{{splunk.home}}/etc/splunk-launch.conf"
+    regexp: '^JAVA_HOME'
```

Comment:
If there's no regex match, will it still write this line?

Reply:
yes